src/plugins/usage_collection/server/collector/collector_set.ts

```diff
@@ -208,7 +214,15 @@ export class CollectorSet {
             soClient,
             ...(collector.extendFetchContext.kibanaRequest && { kibanaRequest }),
           };
-          const result = await collector.fetch(context);
+          const executionContext: KibanaExecutionContext = {
+            type: 'usage_collector',
+            name: 'telemetry_usage_collector', // public name of a user-facing feature
```

Comment:
I wonder if we should use an alternative name to `telemetry_*`? In case users see too many execution context logs talking about telemetry, and decide to disable it 😇

Reply:
+1 for not using the `telemetry` term here

Reply:
Done!